week02/group_anagrams.cpp

```diff
@@ -0,0 +1,26 @@
+//Problem 6: Group anagrams. (Difficulty: Medium)
+
+//Solution:
+
+// Time complexity: O(N), where N is number of strings.
+// Space complexity: O(N)
+
+unsigned long long getHash(string& str){
+    unsigned long long hashValue = 1;
+    for(auto ch: str){
+        hashValue = hashValue * (257 + (ch - 'a'));
+    }
+    return hashValue;
+}
```

Comment:
It's better not to use `unsigned long long`. Refer to my [this](https://discord.com/channels/936330694689447978/958964922971217931/970277008183541770) post.